test/e2e/apimachinery/aggregator.go

```diff
@@ -73,7 +73,12 @@ var _ = SIGDescribe("Aggregator", func() {
 		aggrclient = f.AggregatorClient
 	})
 
-	It("Should be able to support the 1.7 Sample API Server using the current Aggregator", func() {
+	/*
+		    Testname: aggregator-supports-the-sample-apiserver
+		    Description: Ensure that the sample-apiserver code from 1.7 and compiled against 1.7
+			will work on the current Aggregator/API-Server.
+	*/
+	framework.ConformanceIt("Should be able to support the 1.7 Sample API Server using the current Aggregator", func() {
```

Comment:
Why is this release-specific?

Reply:
This tests against an old image based on 1.7 k8s.io/apiserver, i.e. it's a kind of version skew test.

The follow-up question: what are our API guarantees here? Will we support aggregated apiservers built against a 1.7 k8s.io/apiserver forever? If not, how far back do we go?

Reply:
Also note that maximal client and aggregated server skew might add up: client-go +1, aggregated server -2, makes effectively +3 for the client.